streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/SettingServiceImpl.java

```diff
@@ -100,7 +105,31 @@ public MavenConfig getMavenConfig() {
 
     @Override
     public DockerConfig getDockerConfig() {
-        return DockerConfig.fromSetting();
+        List<Setting> settingList = settingMapper.querySettingByKeys(Arrays.asList(
```

Comment:
baseMapper.querySettingByKeys(xxx)

Reply:
I modified it. However, I feel that using baseMapper here will reduce the readability and maintainability of the code, since other mapper may be added to the service in the future.

Reply:
In AService, we can only call AMapper (baseMapper),  and cannot call BMapper directly(we can call BService...)